mlflow/tracking/_tracking_service/client.py

```diff
@@ -359,14 +405,26 @@ def update_run(self, run_id, status=None, name=None):
             run_name=name,
         )
 
-    def log_batch(self, run_id, metrics=(), params=(), tags=()):
+    def log_batch(
+        self, run_id, metrics=(), params=(), tags=(), synchronous: bool = True
+    ) -> RunOperations:
         """
         Log multiple metrics, params, and/or tags.
 
         :param run_id: String ID of the run
         :param metrics: If provided, List of Metric(key, value, timestamp) instances.
         :param params: If provided, List of Param(key, value) instances.
         :param tags: If provided, List of RunTag(key, value) instances.
+        :param synchronous: [Experimental] Defaults to True with no behavior change.
```

Comment:
docstring of `synchronous` is neither vertical aligned or indented with 4 space. Please choose either style. I strongly recommend 4-space over vertical alignment though.

Reply:
I checked on my side again and it appears to be fine, may be I am not getting what you are pointing at, can you please elaborate?


![image](https://github.com/mlflow/mlflow/assets/109864/3a3c2d22-41b2-415b-a6f9-89df976b4153)